### ISSUE TITLE
replace d-hidden with aria-hidden

### ISSUE THREAD

**Issue by wkeese** [ibm-js/delite#231]
See https://www.w3.org/TR/wai-aria-1.1/#aria-hidden.

Probably instead of having our own d-hidden flag we should just use aria-hidden.    Otherwise it's an accessibility violation (unless we use both attributes).

**Comment by wkeese** [comment]
On second thought, there are times (most of the time?) where we want to set aria-hidden on displayed elements... things like icons next to labels, where the labels are sufficient for the screen readers.   So, let's keep both flags.
